etc/config-template.toml

```diff
@@ -40,6 +40,16 @@
 # max recursion level allowed when decoding dag expression
 # end-point-recursion-limit = 1000
 
+# enable or disable dist-sql cache feature
+# enable-distsql-cache = false
+
+# dist-sql cache size default is 256MB.
+# distsql-cache-size = "256MB"
+
+# dist-sql cache entry max size, if result set is bigger than this, dist-sql
```

Comment:
The max size of the dist-sql cache. If the result set is bigger than the value, the dist-sql cache does not cache it.